### PR TITLE
Offline calibration refactor docs update

### PR DESCRIPTION
**Pull request by papr**

### CODE REVIEW

File: user-docs/pupil-player.md [hunk 1]
```diff
@@ -145,50 +145,78 @@ This plugin can be used to overlay the eye video on top of the world video. Note
...
-The `Detection Method` selector sets the detection algorithm to either `2d` or `3d` detection (see [the section on Pupil Detection](#pupil-detection) for details). The `Redetect` button restarts the detection procedure. You can use the `Offline Pupil Detector` plugin to debug, improve, and gain insight into the pupil detection process.
+The `Detection Method` selector sets the detection algorithm to either `2d` or `3d` detection (see [the section on Pupil Detection](#pupil-detection) for details).
+The `Redetect` button restarts the detection procedure.
+You can use the `Offline Pupil Detector` plugin to debug, improve, and gain insight into the pupil detection process.
 
-#### Gaze Positions From Recording
-This is the default gaze mapping plugin. This plugin tries to load gaze positions that were created and stored during a Pupil Capture recording. This plugin can not be run in conjunction with  `Offline Calibration`.
+### Gaze Data And Post-hoc Calibration
+By default, Player starts with the `Gaze From Recording` plugin that tries to load gaze positions that were detected and stored during a Pupil Capture recording.
+Alternatively, one can run the calibration process post-hoc.
```
Comment (ckbaumann) [inline user-docs/pupil-player.md]:
```suggestion
Alternatively, one can run the gaze mapping process post-hoc.
```

File: user-docs/pupil-player.md [hunk 2]
```diff
@@ -145,50 +145,78 @@ This plugin can be used to overlay the eye video on top of the world video. Note
...
+The `Offline Calibration` plugin enables you to calibrate, map, and validate gaze post-hoc and is available starting with Pupil Player `v0.9.13`.
+It can be used on any Pupil dataset.
 
-1. `Circle Marker`: This method tries to automatically detect [circular calibration markers](#calibration-methods) within the world video.
-2. `Natural Features`: This method enables you to select reference points by marking (clicking on) known features that correspond to gaze positions within the world video.
+The workflow is separated into three steps, each with its own submenu: Reference Locations, Calibrations, and Gaze Mappers.
 
-Data is calibrated and mapped in sections. Each section has following properties:
+Reference locations are points within the recording that are known to have been fixated by the subject.
```
Comment (willpatera) [inline user-docs/pupil-player.md]: Reference locations are points within the world 
```suggestion
Reference locations are points within the recorded world video that are known to have been fixated on by the participant/subject.
```

File: user-docs/pupil-player.md [hunk 3]
```diff
@@ -145,50 +145,78 @@ This plugin can be used to overlay the eye video on top of the world video. Note
...
 
-1. `Calibration Method`: `Circle Marker` or `Natural Features`, see above
-2. `Calibration Mode`: `2d` uses polynomial regression, or `3d` uses bundle adjustment calibration
-3. `Calibration Range`: Slice of world frame indices that indicates which reference and pupil positions to use for calibration. For example, the calibration range `[500, 701]` will use all reference and pupil positions for calibration that correlate to the world timestamps with indices 500 - 700
-4. `Mapping Range`: Slice of world frame indices that indicates which pupil positions will be mapped to gaze positions.
+1. `Detect Circle Markers in Recording`: This button starts the automatic detection of [circular calibration markers](#calibration-methods) within the world video. The progress is visualized in the plugin's timeline.
+2. `Manual Edit Mode`: When this option is enabled, you can add new locations as well as correct and delete existing ones. There can only be one location per world frame.
+
+As in Capture, one can have more than one calibration per recording.
+A calibration on its own does not result in gaze data, yet.
```
Comment (willpatera) [inline user-docs/pupil-player.md]:
```suggestion
A calibration on its own does not result in gaze data.
```

File: user-docs/pupil-player.md [hunk 4]
```diff
@@ -145,50 +145,78 @@ This plugin can be used to overlay the eye video on top of the world video. Note
...
+It rather contains the required parameters to map pupil to gaze data.
```
Comment (willpatera) [inline user-docs/pupil-player.md]:
```suggestion
Rather, it contains the required parameters to map pupil to gaze data.
```

File: user-docs/pupil-player.md [hunk 5]
```diff
@@ -145,50 +145,78 @@ This plugin can be used to overlay the eye video on top of the world video. Note
...
+Each has the following properties:
+
+- `Name`: Used to correctly select a calibration for each gaze mapper (see below).
+- `Mapping Method`: `2d` uses polynomial regression, or `3d` uses bundle adjustment calibration
```
Comment (willpatera) [inline user-docs/pupil-player.md]:
```suggestion
- `Mapping Method`: `2d` uses polynomial regression, or `3d` uses bundle adjustment calibration.
```

**Comment by willpatera** [comment]
@papr please apply commits to your branch, and we can merge.

**Pull request merged**